lib/providers/notifiers/connection_notifier.dart

```diff
@@ -2,6 +2,12 @@ import 'package:flutter/foundation.dart';
 import 'package:http/http.dart';
 import 'package:seeds/constants/config.dart';
 
+class Endpoint {
```

Comment:
Nothing to do with this PR, just nerding out. 

Kotlin has this thing called a data class. its lovely. 

sealed classes together with data classes are programmers porn. 

https://kotlinlang.org/docs/reference/sealed-classes.html

Reply:
cool